### PR TITLE
🆕 `article` - CSS Nesting

### PR DESCRIPTION
**Pull request by argyleink**
Ready for review
- [x] get final reviews
- [x] add final publish date

### CODE REVIEW

File: site/en/articles/css-nesting/index.md [hunk 1]
```diff
@@ -0,0 +1,704 @@
...
+HTML they target. If the `.nesting` component in the previous example was
+removed from the project, you could delete the entire group instead of searching
+files for related selector instances.
+
+**Nesting can help with:**
+- Organization
+- Reducing file size
+- Refactoring
+
+{% BrowserCompat 'css.nesting' %}
```
Comment (rachelandrew) [inline site/en/articles/css-nesting/index.md:71]: I don't think this is in BCD right now, trying to find out where it ought to go, maybe attached to the `&` selector. We can add it in later though.

File: site/en/articles/css-nesting/index.md [hunk 2]
```diff
@@ -0,0 +1,704 @@
...
+
+Set to arrive in Chrome stable 112 and available to [try in Safari Technical
```
Comment (rachelandrew) [inline site/en/articles/css-nesting/index.md:73]:
```suggestion
Nesting is available from Chrome 112 and also available to [try in Safari Technical
```

Comment (rachelandrew) [reply site/en/articles/css-nesting/index.md:73]: Once we have BCD we can remove this line as then it will be kept up to date.

**Review by rachelandrew — APPROVED** [review]
All looks good, small nit around browser info to commit. I'll try and get BCD done once I've checked where it should live in the structure.

File: site/en/articles/css-nesting/index.md (hunk 1 above)
Comment (argyleink) [reply site/en/articles/css-nesting/index.md:71]: confirmed it's not, this is a stub. 

i filed a request/issue for it back in jan https://github.com/mdn/mdn/issues/359

agree we can add it later. 

```suggestion
<!-- {% BrowserCompat 'css.nesting' %} -->
```

File: site/en/articles/css-nesting/index.md (hunk 2 above)
Comment (argyleink) [reply site/en/articles/css-nesting/index.md:73]: yep, this was added assuming the BCD data wouldnt be ready

**Comment by argyleink** [comment]
> All looks good, small nit around browser info to commit. I'll try and get BCD done once I've checked where it should live in the structure.

thanks for reviewing! should i [wait for the bcd data](https://github.com/mdn/mdn/issues/359), or just set myself a task to come back to this post later?

**Comment by rachelandrew** [comment]
Fine to merge it as it can take a while for BCD to come through.

File: site/en/articles/css-nesting/index.md [hunk 3]
```diff
@@ -0,0 +1,704 @@
+---
+layout: layouts/blog-post.njk
+title: CSS Nesting
+description: >
+  One of our favorite CSS preprocessor features is now built into the language: nesting style rules.
+subhead: >
+  One of our favorite CSS preprocessor features is now built into the language: nesting style rules.
+date: 2023-03-07
```
Comment (argyleink) [inline site/en/articles/css-nesting/index.md:8]:
```suggestion
date: 2023-03-08
```

update date

**Pull request merged**